jetty-server/src/main/java/org/eclipse/jetty/server/handler/InetAccessHandler.java

```diff
@@ -188,13 +188,16 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
     protected boolean isAllowed(InetAddress addr, Request baseRequest, HttpServletRequest request)
     {
         String name = baseRequest.getHttpChannel().getConnector().getName();
+        boolean includedConnectorName = _names.isIncluded(name);
+        boolean excludedConnectorName = _names.isExcluded(name);
+        boolean filterAppliesToThisConnector = (_names.includesSize() == 0 || includedConnectorName) && !excludedConnectorName;
```

Comment:
The `_names.isIncludedAndNotExcluded(name);` from below is what this block of code does.